type.go

```diff
@@ -117,6 +117,18 @@ func typeFromClangType(cType clang.Type) (Type, error) {
 		// TODO Does not exist in Go, what should we do with it? https://github.com/go-clang/gen/issues/50
 		typ.CGoName = "void"
 		typ.GoName = "void"
+	case clang.Type_IncompleteArray:
+		// do smth intelligent
+		subTyp, err := typeFromClangType(cType.ArrayElementType())
```

Comment:
This is the same code as with Type_ConstantArray except the ArraySize part, which calls cType.ArraySize(), however this call could and should be fine even with an incomplete array. Please merge them together.